binaries/go-http-daemon/main.go

```diff
@@ -62,7 +63,27 @@ func executeRequest(c echo.Context) error {
 		return parseRequestError
 	}
 
-	requestResponsePair, responseErr := request.ExecuteRequest(daemonRequest.ToRequest(), daemonRequest.Profiles, daemonRequest.Variables)
+	var req request.Request
+	if daemonRequest.RequestName != "" {
+		requestOptions, err := profile.LoadRequestOptions(daemonRequest.RequestName, daemonRequest.Profiles)
+
+		if err != nil {
+			return err
+		}
+
+		req = request.Request{
+			Body:    requestOptions.Body,
```

Comment:
Could you use [a library like `deepcopier`](https://github.com/ulule/deepcopier) to simplify this?

Reply:
Probably. I did a fast search and didn't think it was such a big deal. But I'm feeling like I've been copying objects more and more, so will probably do that later.